core/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # @commercetools-test-data/core
 
+## 3.1.0
+
 ## 3.0.2
 
 ### Patch Changes
```

Comment:
FYI: this is how the changelog will look using "fixed" mode for packages that didn't have any change.

cc @commercetools/shield-team-fe @tdeekens